playbooks/openshift/roles/cbs/tasks/install_required_pkgs.yml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
Why not just use the common task here and update the defaults/main.yml to also include centos_packager?